runtime/src/lib.rs

```diff
@@ -292,7 +290,7 @@ construct_runtime!(
 );
 
 /// The address format for describing accounts.
-pub type Address = AccountId;
+pub type Address = sp_runtime::MultiAddress<AccountId, ()>;
```

Comment:
Is it possible that there could be compatibility issues with the current running node? I assume the answer is 'no,' given that we don't have any storage yet, but I want to confirm just to be sure

Reply:
Good point!

The **runtime that I'm tweaking is not installed in the Evochain**. The integration of the Ethereum signature was more of a `wannabe` feature that was **never deployed**.

So, to clarify, the answer to your concern is indeed **"Yes."** However, **this Pull Request (PR) addresses and removes the problem**, ensuring that the system's integrity is maintained without the need for the undeployed feature.